assets/docs/fragments/cli-installation.md

```diff
@@ -0,0 +1,21 @@
+## CLI Installation
+<details>
+<summary>MacOS</summary>
+
+You can install in MacOS via brew: `brew install asyncapi`.
+</details>
+
+<details>
+<summary>Linux</summary>
+
+You can install in Linux via `dpkg`, a debian package manager:
+
```

Comment:
@derberg I had the wrong character that was causing my markdown issue. I must have been too tired yesterday to figure it out 😂 

anyways, this works fine now 👍